libsql_graph_db/models.py

```diff
@@ -1,3 +1,40 @@
 """
 Pydantic models for pythonic API to the graph db
 """
+
+from pydantic import BaseModel, Field
+from typing import Optional, List
+
+
+class Node(BaseModel):
```

Comment:
NOTE: these types will also be used in `graph.py` when wrapping the functions provided by `database.py` into a higher level API